workflowai/core/client/_fn_utils_test.py

```diff
@@ -113,3 +114,58 @@ async def test_fn_stream_output_only(self, mock_api_client: Mock):
         assert len(chunks) == 1
         assert isinstance(chunks[0], HelloTaskOutput)
         assert chunks[0] == HelloTaskOutput(message="Hello, World!")
+
+
+def test_clean_docstring():
+    # Test empty docstring
+    assert clean_docstring("") == ""
```

Comment:
nit: use a pytest parametrize with `value, expected`

Having a single test makes it harder to find where exactly it failed + makes it impossible to retry a single case